usaspending_api/submissions/test_load_submission.py

```diff
@@ -0,0 +1,43 @@
+from django.test import TestCase, Client
+from django.core.management import call_command
+from usaspending_api.accounts.models import *
+from usaspending_api.awards.models import *
+from usaspending_api.financial_activities.models import *
+from usaspending_api.submissions.models import *
+
+import pytest
+import os
+
+from model_mommy import mommy
+
+
+class LoadSubmissionTest(TestCase):
+    def setUp(self):
+        SubmissionAttributes.objects.all().delete()
```

Comment:
I wanted to make sure everything was gone before loading to ensure that it has the appropriate number for the assert statements